src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -2571,15 +2583,11 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     new Comment
                     {
                         Id = new Guid("514940e6-3b84-4e1b-aa5d-d1e5fa671e1b"),
-                        IContentBlockId = new Guid("a0b85d7d-a9bd-48b5-82c6-a119adc74ca2"),
-                        CommentText = "Test Text",
-                        Name = "A Test User",
-                        State = CommentState.open,
-                        Time = new DateTime(2019, 12, 1, 15, 0, 0),
-                        ResolvedBy = null,
-                        ResolvedOn = null
+                        ContentBlockId = new Guid("a0b85d7d-a9bd-48b5-82c6-a119adc74ca2"),
+                        Content = "Test Text",
+                        Created = new DateTime(2019, 12, 1, 15, 0, 0),
+                        CreatedById = new Guid("b99e8358-9a5e-4a3a-9288-6f94c7e1e3dd"),
```

Comment:
I think our user ids are named variables in this file if you wanted to use that instead of duplicating the id here - I could very well be misremembering though :)

Reply:
Yes there is :) This should be fixed now 👍